deployer/src/deployer/main.py

```diff
@@ -1,101 +1,99 @@
-import functools
-import pkg_resources
 from pathlib import Path
 
 import click
 
+from . import __version__
+from .aws_lambda import update_all as update_all_lambda_functions
 from .constants import (
-    DEFAULT_NAME,
-    DEFAULT_NAME_PATTERN,
-    DEFAULT_BUCKET,
-    S3_DEFAULT_BUCKET_LOCATION,
+    CONTENT_ROOT,
+    CONTENT_TRANSLATED_ROOT,
+    DEFAULT_BUCKET_NAME,
     DEFAULT_NO_PROGRESSBAR,
 )
-from .exceptions import CoreException
-from .upload import upload_site
-from .utils import error, info
-from . import __version__
-
-
-def cli_wrap(fn):
-    @functools.wraps(fn)
-    def inner(*args, **kwargs):
-        try:
-            fn(*args, **kwargs)
-        except CoreException as exception:
-            info(exception.__class__.__name__)
-            error(str(exception))
-            raise click.Abort
-
-    return inner
+from .upload import upload_content
+from .utils import log
 
 
 @click.group()
-@click.option("--debug/--no-debug", default=False)
+@click.option(
+    "--dry-run",
+    default=False,
+    help="Show what would be done, but don't actually do it.",
+    show_default=True,
+    is_flag=True,
+)
 @click.version_option(version=__version__)
 @click.pass_context
-def cli(ctx, debug):
+def cli(ctx, **kwargs):
     ctx.ensure_object(dict)
-    ctx.obj["debug"] = debug
+    ctx.obj.update(kwargs)
 
 
 @cli.command()
 @click.pass_context
-@cli_wrap
+def update_lambda_functions(ctx):
+    log.info(f"Deployer ({__version__})", bold=True)
+    update_all_lambda_functions(dry_run=ctx.obj["dry_run"])
+
+
+@cli.command()
 @click.option(
     "--bucket",
-    default=DEFAULT_BUCKET,
-    help=f"Name of the bucket (default {DEFAULT_BUCKET!r})",
-)
-@click.option(
-    "--name",
-    default=DEFAULT_NAME,
-    help=f"Name of the site (default {DEFAULT_NAME_PATTERN!r})",
+    help='Name of the S3 bucket or one of "dev", "stage", or "prod"',
```

Comment:
The `dev`, `stage`, and `prod` options are bucket nicknames or shortcuts for `mdn-content-dev`, `mdn-content-stage`, and `mdn-content-prod` (they're converted to the real bucket name in `upload.py`). Just a convenience since they're the names we'll be using 99% of the time. I'll change the code to indicate that these are special nicknames.

Reply:
We discussed this and agreed that it's a bit too magical to use nick names. Instead, let's keep it dumb and spell out the whole bucket name in full.